File: Programação/Back-End PHP/02-poo/04-mundocao/construtor/Poddle.php
<?php

    class Poddle extends Cachorro {

        private string $raca = "Poddle";

        public function __construct($nome) 
        {
            parent::__construct($nome, $this->raca);
        }

        public function brincar() {
            print("Estou brincando..." . PHP_EOL);
        }

        public function Latir() {
            print("Au au au au au ! Au!" . PHP_EOL);
        }
    }

?>

File: programacao/java/exemploAula/ExemploEncapsulamento/src/cachorro/Poddle.java
package cachorro;
public class Poddle extends Cachorro {

    // Construtor
    public Poddle(String raca, int idade, String nome, String corOlhos) {
        super(raca, idade, nome, corOlhos);
    }

    void brincar() {
        System.out.println("Brincando...");
    }

}
